docs/answers-react-components.renderentitypreviews.md

```diff
@@ -15,5 +15,5 @@ export declare type RenderEntityPreviews = (autocompleteLoading: boolean, vertic
 
 ## Remarks
 
-An onSubmit function is provided to allow an entity preview to be submitted.
+For the entity previews to be navigable in the search bar's dropdown section, wrap each entity preview in a [DropdownItem()](./answers-react-components.dropdownitem.md) component.

```

Comment:
It looks like only the last remark block is added to the docs, so we should include all of the remarks with only one tag